public/resources/remix/index.js

```diff
@@ -83,10 +83,15 @@
   }
 
   function injectStyleSheets(metadata) {
-    var stylesheets =
-      "<link href=\"" + metadata.host + "/resources/remix/clean-slate.min.css\" rel=\"stylesheet\">\n" +
-      "<link href=\"" + metadata.host + "/resources/remix/style.css\" rel=\"stylesheet\">\n";
-    document.head.appendChild(stylesheets);
+    var stylesheet1 = document.createElement("link");
```

Comment:
if you use `var styleheet` here, you can simply reuse the `stylesheet` variable for the second sheet, too. Once you've added the first to the document head, `stylesheet` can be freely reassigned.